.buildkite/pipeline-release-helm.yml

```diff
@@ -36,7 +38,9 @@ steps:
         commands:
           - buildkite-agent artifact download "bin/releaser" /usr/local/
           - chmod u+x /usr/local/bin/releaser
-          - releaser --env=dev --excludes=eck-operator --dry-run=\$DRY_RUN
+          - releaser --env=dev --charts-dir=deploy/eck-stack --dry-run=\$DRY_RUN
+
+      - wait
```

Comment:
Yes but we don't really need it.

Two types of conflicts can happen when updating the Helm chart index:
- two releases runs sequentially or in // (no matter) and the second does 'helm repo update' which updates an old version of the index due to the CDN cache, not the recent version updated by the first release
- two releases run simultaneously and they get, update and download the index at the same time

We hit the first which is fixed by managing dependencies locally hence stopping to 'helm repo update'.
The second is handled in this PR by the use of the GCS generation-match precondition.

This change to run the two releases sequentially adresses the second though we also have the code that prevents it.
This is me being a bit paranoiac. Given a release takes ~1 second, there is no real benefit to run the two in parallel.
So I thought, let's run them sequentially as it could cause some other negative effect that we didn't see.